dev/sparktestsupport/modules.py

```diff
@@ -781,6 +740,57 @@ def __hash__(self):
     ],
 )
 
+
+pyspark_connect = Module(
+    name="pyspark-connect",
+    dependencies=[pyspark_sql, pyspark_ml, connect],
+    source_file_regexes=[
+        "python/pyspark/sql/connect",
+        "python/pyspark/ml/connect",
```

Comment:
@HyukjinKwon I made an additional change, make `pyspark-connect` depends on `pyspark_ml` and then move ml tests here. since I think it maybe not a good idea to make `pyspark_ml` depends on `connect`